src/components/v5/shared/CardWithBios/CardWithBios.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { FC } from 'react';
+
+import { useUserByNameOrAddress } from '~hooks';
+import { AnyExtensionData, InstalledExtensionData } from '~types';
+
+import UserAvatarPopover from '../UserAvatarPopover/UserAvatarPopover';
+import { splitWalletAddress } from '~utils/splitWalletAddress';
+import { useGetInstalledByData } from '~common/Extensions/SpecificSidePanel/partials/hooks';
+import Icon from '~shared/Icon';
+import BurgerMenu from '../BurgerMenu/BurgerMenu';
+import CardPermissions from './partials';
+import UserStatusComponent from './partials/UserStatus';
+import { CardWithBiosProps } from './types';
+
+const displayName = 'v5.shared.CardWithBios';
```

Comment:
`const displayName = 'v5.CardWithBios';`

The 'shared' directory didn't need a prefix in displayName. Please check other components.